packages/data-context/src/sources/ProjectDataSource.ts

```diff
@@ -217,7 +217,7 @@ export class ProjectDataSource {
 
     const MINIMATCH_OPTIONS = { dot: true, matchBase: true }
 
-    const { specPattern = [], excludeSpecPattern = [] } = await this.ctx.project.specPatternsForTestingType(this.ctx.currentProject, this.ctx.coreData.currentTestingType)
+    const { specPattern = [], excludeSpecPattern = [] } = await this.ctx.project.specPatterns()
```

Comment:
If we don't remove the ignoreSpecPattern config option, shouldn't it be taken into account here?